CHANGES.md

```diff
@@ -1,3 +1,6 @@
+## Vaadin Core Elements 0.3.0-beta... (...)
+- Added Core word, so as the bundle includes all the set of Vaadin Components licensed as Apache 2.0.
```

Comment:
Needs rewording, should not say “Vaadin Components” for instance.

Here’s my suggestion:
The product and related releases are renamed to “Vaadin Core Elements”, which is the collection of custom elements (i.e. Web Components) that are licensed under the Apache License 2.0. The term “Vaadin Elements” refers to all custom elements created by Vaadin from now on, including commercial elements.